test/controllers/test_case_controller.py

```diff
@@ -21,6 +23,12 @@ def test_get_all_business_ids_for_collection_exercise(self, mock_engine):
         test_output = case_controller.get_all_business_ids_for_collection_exercise(EXERCISE_ID)
         self.assertEqual(expected_output, test_output)
 
+    def test_get_all_business_ids_for_collection_exercise_failure(self):
```

Comment:
What kind of failure? A more descriptive name would be useful for future devs

Reply:
I have changed the names of the tests to refer to the SQLAlchemyError.